src/lib/modules/Base/Base.spec.js

```diff
@@ -0,0 +1,77 @@
+import BaseModule from '../Base';
+
+describe('BaseModule', () => {
+  let module;
+  let filterFunc;
+  let io;
+  beforeEach(() => {
+    io = { emit: jest.fn() };
+    filterFunc = jest.fn(() => true);
+    module = new BaseModule('Carapuce', [filterFunc, filterFunc, filterFunc]);
+  });
+
+  it('should have Carapuce as name attribut', () => {
+    expect(module.name).toBe('Carapuce');
+  });
+
+  it('should set io as attribut when mount is called', () => {
+    module.mount(io);
+    expect(module.io).toBe(io);
+  });
+
+  it('should set io as undefined by default', () => {
```

Comment:
If we don't have `io`, the server can't start, this way, the library doesn't have any sense 😅 

In my opinion, we should `throw` an error if it's not set, it will:

- guarantee that the lib is working / not working and why
- reduce the number of LOCs to avoid specific checks like `if (this.io) {}`